articles/what-is-ransomware/index.md

```diff
@@ -0,0 +1,50 @@
+
+### How does it operate?
+Ransomware uses something called cryptoviral extortion to operate, which is a three step process:
+- Attackers generate a pair of cryptographic keys––a public and a private––and place the public key in the malware. The private key remains solely with the attacker.
+- When released onto a device, the malware generates a random symmetric key to encrypt the victim’s data. The public key is then used to encrypt the symmetric key. This process is called hybrid encryption, and creates an asymmetric ciphertext of the victim’s data. It also erases the symmetric key and the victim’s original plaintext data. The only way a victim can recover their data is by paying the ransom and sending the asymmetric ciphertext to the attacker––which is part of the message the user sees.
+- Once the attacker receives their payment and the asymmetric ciphertext, they decrypt the data using their private key, thus successfully completing the attack.
+
+Ransomware is the type of malware that conceals its true intent––by masking itself as a “warning from the authorities.” In computing, such type of malware is called Trojan, since it misleads users by appearing unsuspicious.
+
+### What should you do if infected?
+Here are some things you should do if you believe your device is infected:
+- First and foremost, do NOT pay the ransom.
+- Disconnect from all networks––unplug any ethernet cables, disconnect from WiFi, turn off your bluetooth, go into airplane mode.
+- Disconnect all external devices such as USB/memory sticks, phones, cameras, or external hard drives.
+- Report the incident––depending on where it occurred/on which device. If on a personal device, try the local authorities or even the FBI Internet Crime Complaint Center (IC3). If on campus, using a campus device, report it to campus security or any other authority figure.
+
+### How can you protect yourself?
+Instead of having to deal with the messy aftermath of a ransomware attack, you can take preventative measures to make sure you and your devices remain protected from such threats in the first place. Here are some things you can do:
+- If met with suspicious or unsolicited emails, do not open links or attachments within them, and do not approve or initiate downloads.
+- This next step is important, no matter how tedious it may sound. Make a habit of regularly backing up your crucial data and information onto either an external hard drive or a USB stick. Also remember to securely disconnect said device once the backup is complete. Cloud storage is also an option––but make sure to use a server with high level encryption and multiple-factor authentication.
+- Keep your system updated to the latest version, older versions of software may be more vulnerable to cyber attacks.
```

Comment:
Comma splice. "Old versions..." is the start of a new sentence. Semi-colon or period required.